js/clinicalTimeline.js

```diff
@@ -261,6 +284,121 @@ window.clinicalTimeline = (function(){
   }
 
   /**
+   * Hanles the drawing and panning of the overviewAxis
+   */
+  function handleOverviewAxis() {
+    var tickTemp = getTickValues(minDays, maxDays, "months");
```

Comment:
`tickTemp` what does this variable represent? In general variables named `temp` are frowned upon unless they track temperature. There are a few situations where it can make sense to have it represent "temporary", but it's best to avoid.